forge/src/main/java/dev/architectury/registry/level/biome/forge/BiomeModificationsImpl.java

```diff
@@ -136,6 +139,11 @@ public Optional<ResourceLocation> getKey() {
             public BiomeProperties getProperties() {
                 return properties;
             }
+            
+            @Override
+            public boolean hasTag(TagKey<Biome> tag) {
+                return GameInstance.getServer().registryAccess().registryOrThrow(Registry.BIOME_REGISTRY).getHolder(biomeResourceKey.get()).get().is(tag);
```

Comment:
Would it be wise here to check if the `getHolder` optional is null?

Reply:
Maybe, but it'll error out anyway, so I don't see the point. The chances of it being null are near zero.